spec/v2/swagger.yaml

```diff
@@ -1393,6 +1470,8 @@ components:
           type: object
         type:
           description: The type of the widget
+          enum:
```

Comment:
Please remove this two lines, otherwise we will have troubles with dashboards

Reply:
done, also submitted changes